README.md

```diff
@@ -186,7 +186,7 @@ Here is an example of a peer with `TLS` enabled, please note the `pem` property
 ```
 
 ##### Associating identities with nodes
-After creating an environment the next step to enable a connection is to associate each node with an identity. To do this click on the name of the environment you just created in the `Fabric Environments` panel. You wil then see each of the nodes that need some setup. Click on each one to associate an identity with that node. The identity must be an admin identity for the node.
+After creating an environment the next step to enable a connection is to associate each node with an identity. To do this click on the name of the environment you just created in the `Fabric Environments` panel. You wil then see each of the nodes that need some setup. Click on each one to associate an identity with that node. The identity must be an admin identity for the node. If you want to change which identity is associated with a node then when you are connected to an environment, right click on the node you wish to change, then select `Replace identity associated with a node`.
```

Comment:
"After creating an environment, the next step before connecting is to associate an identity with each node. To do this, click the name of the environment you have just created in the `Fabric Environments` panel. To complete setup, click on each node from the list to associate an identity with them. The identity must be an admin identity for the node. To change the identity associated with a node when connected the environment, right-click on the node and  select `Replace identity associated with a node`." - Few changes in there.